app/src/main/java/com/mobven/fitai/login/sign_up/screens/DailyMovementFragment.kt

```diff
@@ -0,0 +1,70 @@
+package com.mobven.fitai.login.sign_up.screens
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.viewpager2.widget.ViewPager2
+import com.mobven.fitai.R
+import com.mobven.fitai.databinding.FragmentDailyMovementBinding
+import com.mobven.fitai.login.sign_up.adapter.SignUpListAdapter
+import com.mobven.fitai.login.sign_up.model.SignUpSelectorItem
+import com.mobven.fitai.util.SignUpSelectorType
+
+
+class DailyMovementFragment : Fragment() {
+
+    private lateinit var binding: FragmentDailyMovementBinding
+    private var adapter = SignUpListAdapter()
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = FragmentDailyMovementBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val dailyMovementList = listOf(
```

Comment:
Bu listeleri viewModel'de oluşturabilirsin. Fragment oldukça sade olmalı. Mock datalar için mock repository ya da object oluşturabilirsin, viewModel de mockRepository ya da object'e erişir.

Reply:
ViewModel, Repository kısımlarını ve sadeleştirmeleri biraz daha ilerleyen zamanda yapmayı düşünüyoruz, en azından back tarafından bir endpoint bize döndüğü zaman.